examples/collector-exporter-node/package.json

```diff
@@ -34,7 +34,8 @@
     "@opentelemetry/exporter-collector-grpc": "^0.11.0",
     "@opentelemetry/exporter-collector-proto": "^0.11.0",
     "@opentelemetry/metrics": "^0.11.0",
-    "@opentelemetry/tracing": "^0.11.0"
+    "@opentelemetry/tracing": "^0.11.0",
+    "grpc": "^1.24.2"
```

Comment:
Why is this dependency added with no other code changes?

Reply:
was testing creating metadata etc. and needed th grpc to be already installed.